src/transformStateWithClones.js

```diff
@@ -1,13 +1,41 @@
 'use strict';
 
 /**
- * @param {Object} state
+ * @param {Object} stateClone
  * @param {Object[]} actions
  *
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateArray = [];
+  let stateClone = { ...state };
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
```suggestion
  for (const action of actions) {
```

We don't need indexes here